packages/airnode-adapter/e2e/index.ts

```diff
@@ -274,4 +274,9 @@ describe('Extraction, encoding and simple on chain decoding', () => {
       );
     });
   });
+
+  it.only('wow', async () => {
```

Comment:
I assume this can be removed?

Reply:
Yeah, sorry. This was incorrectly squashed when I was cleaning up after myself.